### PR TITLE
add aqara QBCZ15LM

### PR DESCRIPTION
**Pull request by z-x-f**

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -6273,6 +6273,26 @@ const converters = {
             }
         },
     },
+    xiaomi_button_switch_config: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty(0x0226)) {
+                const lookup = {0: 'relay', 1: 'relay&usb'};
```
Comment (Koenkk) [inline converters/fromZigbee.js:6281]: Please rename `relay&usb` to `relay_and_usb`.

**Comment by Koenkk** [comment]
Made some changes:
- Merged `fz.xiaomi_overload_protection`, `fz.xiaomi_button_switch_config` and `fz.xiaomi_socket_local_lock` with `fz.xiaomi_switch_opple_basic`
  - `overload_protection` is called `consumer_overload` now
  - `button_switch_config` -> `button_switch_mode`
- `local_lock`  is a switch now instead of enum
- add overload_protection for ZNCZ04LM
- `local_lock` -> `button_lock`

Can you check if everything still works OK?

**Comment by z-x-f** [comment]
All functions work normally.
I think the <code>button lock</code> is better now.
In addition, the maximum value of  ZNCZ15LM <code>overload_protection</code> in aqara home app is 2500w.

**Comment by z-x-f** [comment]
Can<code>button lock</code> be replaced with other exposes? Now <code>state</code> cannot be Indistinguishable in ha.
I am also going to submit the socket of QBCZ14LM. The button lock is not provided in the aqara home app, but through H1, I found that this function is also available in QBCZ14LM and ZNCZ15LM. However, in QBCZ14LM and ZNCZ15LM, <code>on_off</code> has already used <code>state</code>, if <code>button lock</code> also uses <code>state</code>, ha cannot distinguish between <code>on_off</code> and <code>button lock</code>.

**Comment by Koenkk** [comment]
>  Now state cannot be Indistinguishable in ha.

What do you mean with this? The name in HA?

**Comment by z-x-f** [comment]
HAHA, forgive my bad english. 
In homeassistant, a device has only one <code>state</code>. Now, <code>on/off</code> occupies it. If the <code>button lock</code> also uses state, then they will be confused in the homeassistant.

**Comment by Koenkk** [comment]
This should work fine, there are more devices exposing multiple switches

**Comment by z-x-f** [comment]
Now <code>switch.outlet_masterbedroom_t1</code> in homeassistant cannot distinguish between <code>on/off</code> and <code>button lock</code>


The <code>button lock</code> of H1 is<code>switch.outlet_masterbedroom_h1</code>, <code>on/off</code> is <code>switch.outlet_masterbedroom_h1_ralay</code> and <code>switch.outlet_masterbedroom_h1_usb</code>.

**Comment by z-x-f** [comment]
@Koenkk  Added an exposes to solve the problem of entities in homeassistant, please review to make sure there is no problem.

**Comment by Koenkk** [comment]
Ah, yes it should have been binary instead of switch, thanks!

**Pull request merged**